### PR TITLE
Services group

### PR DESCRIPTION
**Pull request by karenmrose**
## Description
This PR adds the full list of services and formats it correctly. In order to make the code reusable, scalable, and give us the ability to format it according to the designs, I created several components:

Services > Services Group > Service Fields > Services Checkbox

where:
Services = All of the Service related code for the form
Services Group = each group of services with appropriate heading
Service Fields = the list of the fields in each group
Services Checkbox = the formatted checkbox for each service

## Test Plan
1. Load the app
1. Go to the services section and make sure it matches the designs. There should be 6 groups of services.
1. Click Verify Next Resource until you load a resource that has services. Make sure the correct services are automatically selected.
1. Click a couple services for a resource and save. Make sure they are successfully saved.

### CODE REVIEW

**Comment by katelynsills** [comment]
I'm seeing this warning in the console, not sure if it is something to care about or not: `[JSS] Could not find the referenced rule input in ServicesCheckbox`

File: client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js [hunk 1]
```diff
@@ -87,7 +87,8 @@ const populateForm = ({ changeFieldValue, resource }) => {
 		_.get(verifiedData, 'primaryContactPerson.verified'),
 	)
 
-	_.forEach(services, service => {
+	console.log('SERVICES', pregnancyCenterServices)
```
Comment (katelynsills) [inline client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js]: I think we want to remove the log here, right?

Comment (karenmrose) [reply client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js]: Eek good catch!

**Comment by katelynsills** [comment]
I added some comments. I think we need to resolve conflicts, but other than that, this looks FANTASTIC.

**Comment by karenmrose** [comment]
Yay, thanks Kate! I agree, I love how much the services add to the app. Both the number/quality of the list of services and the awesome checkboxes that Marylouise designed! 

Regarding the warning, I found the place that's causing it, but unfortunately removing it causes the hover to not work anymore. It doesn't seem to be causing other issues, so I'd say we leave it for now. I think it's just something related to JSS.

**Pull request merged**